app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -211,11 +211,6 @@ type SelectedPaymentMethod {
 type SelectedPaymentMethodAdditionalData {
 }
 
-enum AdressTypeEnum {
```

Comment:
Please, revert this change
it is better to use the `ENUM` if you know in advance the list of possible options
Thanks

Reply:
Changes were reverted